meta-iotqa/lib/oeqa/runtime/multimedia/audio/pulseaudio_rec_play.py

```diff
@@ -0,0 +1,28 @@
+import time
+from oeqa.oetest import oeRuntimeTest
+
+
+class PulseaudioTest(oeRuntimeTest):
+    def test_rec_play(self):
+        # Start pulseaudio daemon
+        (status, output) = self.target.run("pulseaudio -D")
+        self.assertEqual(status, 0, msg="Error pulseaudio not started: %s" % output)
+        # Recording audio
+        (status, output) = self.target.run("parecord -r /tmp/rec.wav &")
+        time.sleep(3)
+        self.assertEqual(status, 0, msg="Error not recorded: %s" % output)
+        # Stop pulseaudio daemon
+        (status, output) = self.target.run("pulseaudio -k")
+        self.assertEqual(status, 0, msg="Error pulseaudio not stop: %s" % output)
+        # start pulseaudio daemon
+        (status, output) = self.target.run("pulseaudio -D")
+        self.assertEqual(status, 0, msg="Error pulseaudio not started: %s" % output)
+        # Checking recorded file present
+        (status, output) = self.target.run("ls /tmp/ |grep 'rec.wav'")
+        self.assertEqual(status, 0, msg="Error file not found: %s" % output)
+        # Playing audio
+        (status, output) = self.target.run("paplay /tmp/rec.wav &")

```

Comment:
Have you actually tried that you can record and play the audio with these commands (like in your own pc)?